src/trie/merkle_tree.rs

```diff
@@ -296,7 +292,7 @@ pub struct MerkleTree<H: StarkHash> {
     /// The list of nodes that should be removed from the underlying database during the next commit.
     death_row: Vec<TrieKey>,
     /// The list of leaves that have been modified during the current commit.
-    cache_leaf_modified: HashMap<Vec<u8>, InsertOrRemove<Felt>>,
+    pub cache_leaf_modified: HashMap<Vec<u8>, InsertOrRemove<Felt>>,
```

Comment:
If you only read it for reading, maybe a getter will be better, than `pub`

Reply:
ok